otherlibs/unix/getgr.c

```diff
@@ -28,7 +28,8 @@ static value alloc_group_entry(struct group *entry)
 
   Begin_roots3 (name, pass, mem);
     name = caml_copy_string(entry->gr_name);
-    pass = caml_copy_string(entry->gr_passwd);
+    /* on some platforms, namely Android, gr_passwd can be NULL - hence this workaround */
+    pass = caml_copy_string(entry->gr_passwd ? entry->gr_passwd : "");
```

Comment:
Is this worth noting in the ChangeLog explicitly, since it involves password handling? (that a NULL response equates to the empty string)